drivers/shared/executor/executor_linux.go

```diff
@@ -161,7 +161,10 @@ func (l *LibcontainerExecutor) Launch(command *ExecCommand) (*ProcessState, erro
 	if err != nil {
 		return nil, fmt.Errorf("failed to determine relative path base=%q target=%q: %v", command.TaskDir, path, err)
 	}
-	path = rel
+
+	// Turn relative-to-chroot path into absolute path to avoid
+	// libcontainer trying to resolve the binary using $PATH
+	path = "/" + rel
```

Comment:
Do we need to handle the case when rel already starts with "/" ?

Reply:
Going to handle it by using filepath.Join at @nickethier's suggestion below.